en/style-guide.adoc

```diff
@@ -210,13 +210,13 @@ All variable and function names should be lowercase by default, unless there is
 
 == Macros (Red/System) anchor:macros-redsystem[]
 
-Apply the same naming conventions for picking up Red/System macros names. Macros generally use uppercase for names, as a way to visually distinguish them easily from the rest of the code (unless the intention is explicit to make it look like regular code, like pseudo-custom datatype definitions). When multiple words are used, they are separated by an underscore `_` character to increase even more the difference with regular code.
+Apply the same naming conventions when choosing Red/System macros names. Macros are generally uppercase, to visually distinguish them from the rest of the code (unless the intention is to make the macros look like regular code, like pseudo-custom datatype definitions). When multiple words are used, they are separated by an underscore `_` character to increase the difference with regular code even more.
```

Comment:
"Names of macros are generally written in uppercase" sounds better, otherwise it makes one think that the name **and** the body of a macro should be uppercased. Also "Like pseudo-datatype definitions", "to emphasize macro's difference from a regular code".